src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -61,6 +61,19 @@ const AudioInputVFControl: React.FC<Props> = ({
     };
   }, []);
 
+  useEffect(() => {
+    // When user turns on Amazon Voice Focus and the default device changes in Chrome,
+    // we will wrap Amazon Voice Focus to the default device.
+    const wrapVoiceFocus = async () => {
```

Comment:
hmm We should not fix it here. It is only applied to the control (UI). If builders do not use the UI then they have to handle this themselves?

Reply:
It probably needs to be handled in the same place that chooses the device. A better approach would be welcome.

Reply:
Note that the JS SDK has the application do it, too:

https://github.com/aws/amazon-chime-sdk-js/blob/eeb8670b17914a6312810cb24f7b5d727a5fe4f8/demos/browser/app/meetingV2/meetingV2.ts#L1430

Reply:
Yes but in React we have logic to handle this in the library itself (`<Audio/Video>InputProvider`) so its a bit weird to ask whoever use that hook to now handle that logic (which they don't have to do before).

Reply:
Yes, JS SDK doesn't have this issue, since it can select a transform device when the current default device changes while in React, it can just accept a string, which means it can just take a normal device, so we need to add VF to it later.
`currentAudioInputDevice: AudioInputDevice | undefined;` [line 458](https://github.com/aws/amazon-chime-sdk-js/blob/eeb8670b17914a6312810cb24f7b5d727a5fe4f8/demos/browser/app/meetingV2/meetingV2.ts#L458)

Reply:
> hmm We should not fix it here. It is only applied to the control (UI). If builders do not use the UI then they have to handle this themselves?

Yes, it only fix the UI, but there is no problem in our providers. Our meeting provider and audio provider don't recognize transform device, and VF provider is separate from them which is by design. 

We tried to add more logic where handle the "default" device in the `AudioInputProvider` in previous PR, but caused a Sev2, since it would need to wrap `AudioInputProvider` inside `VoiceFocusProvider` in builders' app. If builders don't use the UI, feel like they need to handle this issue themselves...

Reply:
A refactor like that was planned as a follow-up if needed; we prioritized shipping.

Reply:
Yep, discussed with @jing-chen1 and we would write a SIM for this to follow up later so we can ignore this comment for now :)